app/recordtransfer/management/commands/send_test_email.py

```diff
@@ -0,0 +1,156 @@
+import logging
+from datetime import timedelta
+import argparse
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from recordtransfer.emails import (
+    send_submission_creation_failure,
+    send_submission_creation_success,
+    send_thank_you_for_your_submission,
+    send_user_account_updated,
+    send_user_activation_email,
+    send_user_in_progress_submission_expiring,
+    send_your_submission_did_not_go_through,
+)
+
+from recordtransfer.models import InProgressSubmission, Submission, UploadSession, User
+
+
+logger = logging.getLogger(__name__)
+
+EMAIL_FUNCTIONS = {
+    "submission_creation_success": send_submission_creation_success,
+    "submission_creation_failure": send_submission_creation_failure,
+    "thank_you_for_your_submission": send_thank_you_for_your_submission,
+    "your_submission_did_not_go_through": send_your_submission_did_not_go_through,
+    "user_activation_email": send_user_activation_email,
+    "user_account_updated": send_user_account_updated,
+    "in_progress_submission_expiring": send_user_in_progress_submission_expiring,
```

Comment:
This element doesn't match the same naming scheme as the others. Can you change this to:

```suggestion
    "user_in_progress_submission_expiring": send_user_in_progress_submission_expiring,
```